src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -1,17 +1,71 @@
 package frc.robot.subsystems;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
+import java.lang.Math;
+
+/**
+ * What does it do? When the A button is pressed- it shoots a ball. What modes
+ * does it have? Semi-Auto and DUMP. what interactions does it have with other
+ * subsystems? Asks the Indexer and 'vision' if it's ready to shoot. How is it
+ * triggered/OI? A semi-auto button A- or X for shoot all. Does it store any
+ * state? No. Sensors? No.
+ */
 
 public class Shooter extends SubsystemBase {
 
-    private SendableSpeedController shooterWheelMotor;
+    private final SendableSpeedController shooterWheelMotor;
+    public static double speed;
+    public static double gravity;
+    public static double theta;
```

Comment:
`gravity` and `theta` should be final and in upper snake case. Also set the values here.